### DISCUSSION TITLE
No option to Enter the code and also Can't see Copilot icon in status bar on Visual studio code

### DISCUSSION THREAD

**Discussion by sukhiboparai** [community/community#7239] [Copilot Conversations]
I got email from github saying I am invited to use github copilot. I am succuafully able to sign in but then I don't see any option to be whitelist and Also there is no option to enter my token in Copilot on visual studio code.

**Comment by spade5** [comment]
I have met the same problem, but it had worked before.

**Comment by WolfspiritM** [comment]
There was no need to enter any code for me. After installing github copilot it asked me to login. I got redirected to github where I have to confirm the login. Afterwards chrome asked me to open a link to vscode. Once I did that I got a notification in vscode asking me to agree to the Telemetry stuff and after agreeing copilot was enabled
